### PR TITLE
Fixed theming for Docs page and some other small improvements

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

File: web/src/stories/Button/Button.mdx [hunk 1]
```diff
@@ -1,8 +1,5 @@
-import { Meta } from '@storybook/addon-docs/blocks'
```
Comment (AleshaOleg) [inline web/src/stories/Button/Button.mdx:1]: Useless stuff, as I'm declaring all of the info in *stories.tsx

File: web/src/stories/Button/button.stories.tsx [hunk 2]
```diff
@@ -12,7 +14,20 @@ export default {
...
+      page: docs,
+      container: ({
+        children,
+        context,
+      }: {
+        children: ReactChild | ReactChildren
+        context: Context<Record<string, unknown>>
+      }): JSX.Element => (
+        <DocsContainer context={context}>
+          <ThemeProvider>{children}</ThemeProvider>
```
Comment (AleshaOleg) [inline web/src/stories/Button/button.stories.tsx:27]: Any ideas about how to provide Theme for all docs files at a single time?

File: web/src/tokens/theo-import.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ theo
   .convert({
     transform: {
       type: 'web',
-      file: '../tokens/grape-skin.yml',
+      file: '../design-tokens/grape-skin.yml',
```
Comment (AleshaOleg) [inline web/src/tokens/theo-import.js:11]: Just renaming `tokens` to `design-tokens` as it was before

File: web/package.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
   "license": "MIT",
   "scripts": {
     "storybook": "yarn tokens:import && NODE_ENV=development yarn tokens:import && start-storybook -s ./ --port=55555",
+    "start": "yarn storybook",
```
Comment (AleshaOleg) [inline web/package.json:10]: Just shortcut

File: web/src/stories/Button/button.stories.tsx (hunk 2 above)
Comment (tobiaslins) [reply web/src/stories/Button/button.stories.tsx:27]: We already provide the  `ThemeProvider` within `preview.js` as decorator!

```js 
addDecorator(story => <ThemeProvider>{story()}</ThemeProvider>)
```

You can remove the complete `container` property from the story!

File: web/.storybook/preview.js [hunk 5]
```diff
@@ -30,3 +31,14 @@ addParameters({
     },
   },
 })
+
+addParameters({
```
Comment (tobiaslins) [inline web/.storybook/preview.js:35]: I just moved the Provider here. Now it's working for docs aswell!

Comment (AleshaOleg) [reply web/.storybook/preview.js:35]: Cool, didn't know how to pass documentation page here. Thanks :)

**Pull request merged**